haas_websocket/rest/haas_rest_handler.py

```diff
@@ -155,27 +134,56 @@ def checkToken(self):
         else:
             return False
 
-    def checkPassword(self):
-        api_password = self.secretManagerGet(self.password_key)
-
-        if (self.api_password == api_password):
-            return True
-        else:
-            logging.info("Updating password from GCP")
-            return False
-
     def tokenUpdate(self):
         self.b_token = self.secretManagerGet(self.bearer_key)
 
         logging.info(f"Updated b_token from GCP")
         return self.b_token
-    
-    # TODO: same as above, fix password updating module
-    # def passwordUpdate(self):
-    #     self.api_password = self.secretManagerGet(self.password_key)
 
-    #     print(f"(INFO) Updated api password from GCP")
-    #     return self.api_password
+    def getOrganizations(self):
```

Comment:
Looks like we need to add some unit tests for the `getOrganizations`, `getThings`, and `getLocations` methods

Reply:
Good point, I think I have those already made in another repo. Let me look for them.